keras_cv/datasets/waymo/transformer.py

```diff
@@ -581,55 +581,36 @@ def build_tensors_from_wod_frame(frame: dataset_pb2.Frame) -> Dict[str, tf.Tenso
 
 
 def build_tensors_for_augmentation(
-    input_frames: List[Dict[str, tf.Tensor]]
+    frame: Dict[str, tf.Tensor]
 ) -> Tuple[tf.Tensor, tf.Tensor]:
     """Builds tensors for multi-frame data augmentation from input frames.
```

Comment:
Please update to reflect this is not multi-frame.

Reply:
Done